src/components/WebexMeeting/WebexMeeting.jsx

```diff
@@ -42,21 +48,29 @@ export default function WebexMeeting({className, meetingID, style}) {
         {isActive
           ? <WebexInMeeting meetingID={ID} />
           : <WebexInterstitialMeeting meetingID={ID} />}
+        {showSettings && (
+          <Modal
+            onClose={() => adapter.meetingsAdapter.toggleSettings(ID)}
+            title="Settings"
+          >
+            <WebexSettings meetingID={ID} />
+          </Modal>
+        )}
+        {showRoster && isActive && (
+          <div className="member-roster-container">
```

Comment:
you can put a class "member-roster" directly on WebexMemberRoster now, before they didn't accept optional classes but now they do

same for control bar

Reply:
done

Reply:
done